onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4644,9 +4644,13 @@ def _aten_native_batch_norm_inference_onnx(
         momentum=momentum,
         training_mode=training,
     )
+
+    # For this function's test case, the returned mean and var must be empty tensors
+    # Seems we don't have any other way to create a real empty tensor, cannot use zero to simulate here
+    empty_int = op.Shape(input, start=0, end=0)
```

Comment:
I suggest
```py
   empty = op.Constant(value = onnx.helper.make_tensor("empty", onnx.TensorProto.FLOAT, [0], []))
```
That will also eliminate the need for the cast.

Unfortunately, there is no shortcut: currently, we can't use something like `[]` since onnxscript doesn't know what type to use. May be we can find some way to support it later.

Reply:
Does constant(value_floats=[]) work?

Reply:
Or do we need 0d

Reply:
A 0d-tensor has size=1, can't be empty.

Reply:
Right, `value_floats=[]` should work, but seems to not work ... seems to produce an int64 tensor, for some reason.

Reply:
I think this may be due to a bug-feature in onnxruntime ... I remember something from a while back. Need to track it down.

Reply:
> Does constant(value_floats=[]) work?

works in eager mode, but failed on check_model() in full graph mode.

Reply:
Changed to:
```
op.Constant(value = onnx.helper.make_tensor("empty", onnx.TensorProto.FLOAT, [0], []))
```

Reply:
I propose that we don’t introduce the onnx import and just wait for the ORT fix to use value_floats, if others don’t think differently.

Reply:
I was mistaken. The issue is in onnxscript (I should have known). It is similar to the reason why plain [] doesn't work, the onnxscript translator doesn't know what (attribute) type to use (in the current implementation). If we use the context information (that is, the opschema info for the op being called, namely "Constant" here), we will be able to translate it correctly.

Anyway, feel free to not make any changes (as Justin suggests) for now, if you don't want to introduce the onnx import.

Reply:
The issue is the same (more or less) as the one discussed here: https://github.com/microsoft/onnxscript/issues/860

Reply:
OK. I will close this PR.